jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -105,6 +105,18 @@ void testGetSchemasWithSchemaPattern(String schemaPattern) throws SQLException {
     }
   }
 
+  /**
+   * Checks that empty result set is returned for invalid schema pattern
+   */
+  @Test
+  void testGetSchemasWithInvalidSchemaPattern() throws SQLException {
```

Comment:
Duplicated with what I added

Reply:
This one will throw exception for getSchemas() as rs is null

```try (ResultSet rs = statement.executeQuery(query)) {```

Reply:
I didn't get the exception, I think it is because of the way I set up the mock tests. I can remove this test